packages/@aws-cdk/aws-elasticsearch/test/domain.test.ts

```diff
@@ -913,6 +913,15 @@ describe('import', () => {
     expect(stack).not.toHaveResource('AWS::Elasticsearch::Domain');
   });
 
+  test('static fromDomainAttributes(attributes) with token arn and endpoint does not throw validation error', () => {
+    const domainArn = new CfnParameter(stack, 'domainArn', { type: 'String' }).valueAsString;
+    const domainEndpoint = new CfnParameter(stack, 'domainEndpoint', { type: 'String' }).valueAsString;
+
+    expect(() => {
+      Domain.fromDomainAttributes(stack, 'Domain', { domainArn, domainEndpoint });
+    }).not.toThrowError(/Invalid URL/);
```

Comment:
I would prefer if the verification in this test was similar to the previous tests (confirming name, ARN, and endpoint are concrete values)

Reply:
Checked equality with the CfnParameters. I'm new to this repo, so not sure how to turn concrete values into Tokens and then use them.

Reply:
yep this is what I wanted!